README.md

```diff
@@ -1,19 +1,36 @@
 # Web MIDI API Polyfill
 
-This JS library is a prototype polyfill and shim for the [Web MIDI API](http://webaudio.github.io/web-midi-api/) (of which Chris is a co-author), using [Jazz-Soft.net's Jazz-Plugin](http://jazz-soft.net/) to enable MIDI support on Windows, OSX and Linux.
-You need to have [version 1.2 or higher](http://jazz-soft.net/download/Jazz-Plugin) of the Jazz-Plugin in order for this polyfill to work properly. This polyfill and the plugin should work on Chrome, Firefox, Safari, Opera and Internet Explorer 10 and 11.
+![nodejs](http://jazz-soft.github.io/img/nodejs.jpg)
+![chrome](http://jazz-soft.github.io/img/chrome.jpg)
+![firefox](http://jazz-soft.github.io/img/firefox.jpg)
+![safari](http://jazz-soft.github.io/img/safari.jpg)
+![opera](http://jazz-soft.github.io/img/opera.jpg)
+![msie](http://jazz-soft.github.io/img/msie.jpg)
+![windows](http://jazz-soft.github.io/img/windows.jpg)
+![macos](http://jazz-soft.github.io/img/macos.jpg)
+![linux](http://jazz-soft.github.io/img/linux.jpg)
 
-This polyfill was originally designed to test usability of the API itself, but it's also useful to enable MIDI scenarios in browsers that don't yet support Web MIDI.
+This javascript library is a prototype polyfill for the [Web MIDI API](http://webaudio.github.io/web-midi-api/) of which Chris is a co-author.
 
-This polyfill now supports multiple simultaneous inputs and outputs, and sending and receiving long messages (sysem exclusive). It also properly dispatches events. Timestamps on send and receive should be properly implemented now, although of course timing will not be very precise on either.
+It was originally designed to test usability of the API itself, but it is currently mainly used as a shim for [Jazz-Soft.net's Jazz-Plugin](http://jazz-soft.net/) to enable MIDI scenarios in browsers that don't yet support Web MIDI.
+
+The polyfill will automatically check to see if the Web MIDI API is already implemented, and if not it will insert itself.
+
+At the moment Chrome (win|osx|linux|android), Opera (win|osx|linux) and the Android WebView component support Web MIDI.
+
+In other browsers you need to have [version 1.2 or higher](http://jazz-soft.net/download/Jazz-Plugin) of the Jazz plugin installed in order for the polyfill to work properly.
+
+The polyfill and the Jazz plugin should work on Chrome (win|osx|linux|android), Firefox (win|osx|linux), Opera (win|osx|linux), Safari (osx, not ios) and Internet Explorer 10 and 11.
```

Comment:
Heh.  I didn't notice.

On Tue, Sep 15, 2015 at 2:06 PM, Sema <notifications@github.com> wrote:

> In README.md
> <https://github.com/cwilso/WebMIDIAPIShim/pull/63#discussion_r39565252>:
>
> >
> > -This polyfill now supports multiple simultaneous inputs and outputs, and sending and receiving long messages (sysem exclusive). It also properly dispatches events. Timestamps on send and receive should be properly implemented now, although of course timing will not be very precise on either.
> > +It was originally designed to test usability of the API itself, but it is currently mainly used as a shim for [Jazz-Soft.net's Jazz-Plugin](http://jazz-soft.net/) to enable MIDI scenarios in browsers that don't yet support Web MIDI.
> > +
> > +The polyfill will automatically check to see if the Web MIDI API is already implemented, and if not it will insert itself.
> > +
> > +At the moment Chrome (win|osx|linux|android), Opera (win|osx|linux) and the Android WebView component support Web MIDI.
> > +
> > +In other browsers you need to have [version 1.2 or higher](http://jazz-soft.net/download/Jazz-Plugin) of the Jazz plugin installed in order for the polyfill to work properly.
> > +
> > +The polyfill and the Jazz plugin should work on Chrome (win|osx|linux|android), Firefox (win|osx|linux), Opera (win|osx|linux), Safari (osx, not ios) and Internet Explorer 10 and 11.
>
> Well, you are the only native English speaker here :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cwilso/WebMIDIAPIShim/pull/63/files#r39565252>.
>